.github/workflows/ci.yml

```diff
@@ -15,9 +15,10 @@ jobs:
     runs-on: ubuntu-22.04
     strategy:
       matrix:
-        stack-version: ["20", "22"]
+        stack-version: ["20", "22", "24"]
```

Comment:
I've added the new Heroku-24 integration test job to the branch protection required checks list:
https://github.com/heroku/heroku-buildpack-go/settings/branch_protection_rules/424628